src/main/webapp/maps.html

```diff
@@ -1,54 +1,59 @@
+    <link rel="stylesheet" href="/css/selectionMenu.css">
+    <link rel="stylesheet" type="text/css" href="/css/navbar.css">
 </head>
 
 <body onload="addLoginOrLogoutLinkToNavigation();buildUI();" id="body" style="overflow:hidden;">
-  <header>
-    <img src="images/popcornIcon.png" alt="Image not available">
-    <nav>
-      <ul id="navigation">
-        <li><a href="/">Home</a></li>
-        <li><a href="/aboutus.html">About Our Team</a></li>
-        <li><a href="/feed.html">Public Feed Page</a></li>
-        <li><a href="/maps.html">Maps Page</a></li>
-      </ul>
-    </nav>
-  </header>
+    <header>
+        <img src="images/popcornIcon.png" alt="Image not available">
+        <nav>
+            <ul id="navigation">
+                <li><a href="/">Home</a></li>
+                <li><a href="/aboutus.html">About Our Team</a></li>
+                <li><a href="/feed.html">Public Feed Page</a></li>
+                <li><a href="/maps.html">Maps Page</a></li>
+            </ul>
+        </nav>
+    </header>
 
-  <div id="myMapsHeader"><h1>
-    Cinema Locations in Singapore
-  </h1></div>
-  <div id="map"></div>
-  <div id="mapsPgPopUp">
-    <div id="firstMenuBody">
-      <button id="closeBtn" onclick="div_hide()">
+    <div id="myMapsHeader">
+        <h1>
+            Cinema Locations in Singapore
+        </h1>
+    </div>
+    <div id="map"></div>
+    <div id="mapsPgPopUp">
+        <div id="firstMenuBody">
+            <button id="closeBtn" onclick="div_hide()">
         <img id="closeBtnImg" src="/images/closeButton_1.png" alt="Index"/>
       </button>
-      <div class="container">
-        <div class="box">
-          <textarea placeholder="Enter your reviews for this cinema here. It can be about the cleanliness, customer service quality, film selection variety, comfort or anything else." id="submitReviewTextArea"></textarea>
-          <div class="content">
-            <div id="cinemaSentimentScore"><h2>36</h2></div>
-            <div id="cinemaName"><h3></h3></div>
-            <p>
-              Submit your Review!
-            </p>
-            <button id="submitReviewsButton">Submit Review</button>
-          </div>
+            <div class="container">
+                <div class="box">
+                    <textarea placeholder="Enter your reviews for this cinema here. It can be about the cleanliness, customer service quality, film selection variety, comfort or anything else." id="submitReviewTextArea"></textarea>
+                    <div class="content">
+                        <div class="sentiment_title">Avg Score:</div>
+                        <div id="cinemaSentimentScore">
+                            <h2></h2>
+                        </div>
+                        <div id="cinemaName">
+                            <h3></h3>
+                        </div>
+                        <button id="submitReviewsButton">Submit Review</button>
```

Comment:
So something like this:

```suggestion
                        <button id="submitReviewsButton"><a>Submit Review</a></button>
```

Reply:
Sorry, I fixed it a few times but I don't know why it keeps getting lost here